src/ucp/core/ucp_ep.h

```diff
@@ -477,8 +482,8 @@ struct ucp_ep_config {
     /* Bitmap of lanes selected by the protocols */
     ucp_lane_map_t                proto_lane_map;
 
-    /* Indicate that protocol initialization was done for config */
-    int                           is_proto_initialized;
+    /* EP initialization flags */
+    ucp_ep_init_flags_t           proto_init_flags;
```

Comment:
unsigned (since it's not really an enum)

Reply:
right, fixed